### DISCUSSION TITLE
How do I access the Downloads directory?

### DISCUSSION THREAD

**Discussion by synsax** [holzschu/a-shell#580] [Q&A]
I can't find the Downloads directory, does anyone know where it is?
When I used the app before, I could access it with ~Downloads as follows, but now I cannot :(
```
[~]$ cd ~Downloads
cd: /private/var/mobile/Containers/Data/Appli
cation/36EBC757-A704-4BE7-9298-3D8B2027A729/~
Downloads: no such file or directory
```

Thanks for providing us with a great app!
As far as I know, it is the number ONE app on iOS that uses shell and can run python!

thank you 👯

**Comment by holzschu** [comment]
I don't think Downloads is part of the default bookmarks. You need to pick it once with `pickFolder`, then it will be.

If something happens that erases the bookmark, you will have to use `pickFolder` again. bookmarks are resilient: they stay valid after a reboot and when a-Shell updates itself (but not when the destination app updates itself, because the destination directory changes). "Downloads" should remain valid a very long time, but I don't know exactly how iOS decides that a bookmark ceases to be valid.
